crt_portal/cts_forms/templates/forms/snippets/error_alert.html

```diff
@@ -0,0 +1,9 @@
+        <div class="usa-alert usa-alert--error usa-alert--slim">
+          <div class="usa-alert__body">
+            <em class="usa-alert__text">
+              {% for error in errors %}
```

Comment:
If we don't iterate over the errors, Django will render a `<ul>` wrapper element which we don't want and which throws off our styles. So this is a style fix.